volatility3/framework/symbols/linux/extensions/elf.py

```diff
@@ -140,36 +146,103 @@ def get_section_headers(self):
         )
         return section_headers
 
+    def get_link_maps(self, kernel_symbol_table_name):
+        """Get the ELF link map objects for the given VMA address
+
+        Args:
+            kernel_symbol_table_name (str): Kernel symbol table name
+
+        Yields:
+            The ELF link map objects
+        """
+        got_entry_size = self._ei_class_size // 8
+
+        elf_symbol_table = self.get_symbol_table_name()
+
+        link_maps_seen = set()
+        for phdr in self.get_program_headers():
+            try:
+                if phdr.p_type.description != "PT_DYNAMIC":
+                    continue
+            except ValueError:
+                continue
```

Comment:
Probably some `LOGLEVEL_VVVV` logs would be useful here, just for anyone trying to figure out why thins aren't happening when they might expect them to...

Reply:
done